CHANGES.rst

```diff
@@ -130,8 +133,6 @@ set_telescope_pointing
 
 - Migrate set_telescope_pointing to pysiaf-based exclusively [#6993]
 
-- Return non-zero status from the set_telescope_pointing command-line when an error occurs [#7056]
```

Comment:
I think this should remain, in case anyone tries to install `1.8.1` and has this issue